javaslang/src/main/java/javaslang/control/GwtIncompatible.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2013 Google Inc.
```

Comment:
This is what I found:

> All of GWT's code is available under the Apache 2.0 open source license.

Then mentioning Google's Copyright is the way to go, right?